### PR TITLE
Gherkin: Added all types of health check 

### PR DESCRIPTION
**Pull request by sanketpathak**
Story: https://issues.redhat.com/browse/ODC-5484

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/healthChecks/add-health-checks-in-topology-page.feature [hunk 1]
```diff
@@ -17,53 +17,99 @@ Feature: Health Checks
...
 
+    Examples:
+        | readinesss_type   | liveness_type     | startup_type      |
+        | HTTP GET          | HTTP GET          | HTTP GET          |
+        | TCP Socket        | TCP Socket        | TCP Socket        |
+        | Container Command | Container Command | Container Command |
+
 
     @regression
     Scenario: Add Startup Probe Health Check to Deployments from Context Menu: MH-05-TC02
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/healthChecks/add-health-checks-in-topology-page.feature:70]: It says only Startup probe. Can you please update the scenario description?

**Review by gajanan-more — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/integration-tests/features/healthChecks/add-health-checks-in-topology-page.feature (hunk 1 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/healthChecks/add-health-checks-in-topology-page.feature]: Updated

**Review by gajanan-more — APPROVED** [review]
/lgtm

**Pull request merged**